run/main.go

```diff
@@ -4,22 +4,25 @@ import (
 	"os"
 
 	"github.com/paketo-buildpacks/mri"
-	"github.com/paketo-buildpacks/packit"
-	"github.com/paketo-buildpacks/packit/cargo"
-	"github.com/paketo-buildpacks/packit/chronos"
-	"github.com/paketo-buildpacks/packit/draft"
-	"github.com/paketo-buildpacks/packit/pexec"
-	"github.com/paketo-buildpacks/packit/postal"
-	"github.com/paketo-buildpacks/packit/scribe"
+	"github.com/paketo-buildpacks/packit/v2"
+	"github.com/paketo-buildpacks/packit/v2/cargo"
+	"github.com/paketo-buildpacks/packit/v2/chronos"
+	"github.com/paketo-buildpacks/packit/v2/draft"
+	"github.com/paketo-buildpacks/packit/v2/pexec"
+	"github.com/paketo-buildpacks/packit/v2/postal"
+	"github.com/paketo-buildpacks/packit/v2/scribe"
 )
 
 func main() {
+	logger := scribe.NewEmitter(os.Stdout)
+	logger.Logger = logger.Logger.WithLevel(os.Getenv("BP_LOG_LEVEL"))
```

Comment:
This construction seems kind of awkward. I think we can possibly do better in `scribe` to make it so that this API seems a bit more fluent. Possibly something like this,

```go
logger := scribe.NewEmitter(scribe.NewLogger(os.Stdout).WithLevel(os.Getenv("BP_LOG_LEVEL"))
```

Reply:
see my comment on the associated packit issue about this

Reply:
fix blocked on https://github.com/paketo-buildpacks/packit/pull/274